lib/methods/comments.js

```diff
@@ -0,0 +1,151 @@
+Meteor.methods({
+  addComment(data) {
+    let { collectionId, content, collectionType } = data;
+
+    let commentId = Comments.insert({
+      ownerId: Meteor.userId(),
+      content: content,
+      createdAt: Date.now()
+    });
+
+    let userId = Meteor.userId();
+
+    if (!commentId)
+      throw new Meteor.Error(500, "Something went wrong, please try again later");
+
+    // FIXME: High priority refactoring task
```

Comment:
all of these notification insertion will not be there in the next release inshaAllah

and yes, we can't keep that into js files anymore